src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +133,95 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            project_globals.NTD_OBSERVER: {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return project_globals.NTD_OBSERVER
+
+    def setup(self, builder):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.config = builder.configuration['metrics'][project_globals.NTD_OBSERVER].to_dict()
+        self.config['by_age'] = False
+
+        tmp = builder.configuration['time']['start']
+        self._sim_start = pd.Timestamp(tmp.year, tmp.month, tmp.day)
+        tmp = builder.configuration['time']['end']
+        self._sim_end = pd.Timestamp(tmp.year, tmp.month, tmp.day)
+
+        columns_required = ['alive', 'dead', f'{self.disease}', 'entrance_time', 'tracked', 'untracked']
```

Comment:
shouldn't have an untracked column.  I'm surprised this doesn't fail.  No untracked column exists.